### DISCUSSION TITLE
Chat Push API

### DISCUSSION THREAD

**Discussion by DeucalionD** [etsy/open-api#970] [Ideas]
I cannot find any API doc mention about chat push API for integration. Just wondering if this is something we can create but hasn't done it yet, or if it is not within Etsy's policy to have this API to integrate chat messages to external apps/websites?

**Comment by Gareth064** [comment]
Hey

this has been discussed before, maybe a couple of years ago, and Etsy said they won't be exposing customer chats/messages via the API.

No harm in asking again. Email them at developer@etsy.com

**Reply by DeucalionD** [reply]
Thanks man!

**Comment by nateswart** [comment]
We are not considering adding chat to the API at this time

**Reply by DeucalionD** [reply]
Thank you for the information,Nate. Do we have any particular plan to build it in the future, or won't be at all? And why, if I may ask.

**Reply by SannanOfficial** [reply]
Are there any workarounds to this without using the API that wouldn't break Etsy's policies?

**Reply by Gareth064** [reply]
@SannanOfficial no there isn't. Chat/Messages are only accessible via a logged in web page. So even scrapping (which breaks Etsy terms) would be near impossible to do.

**Reply by SannanOfficial** [reply]
@Gareth064 Okay, thank you. I went with emailing all the users who purchased a product from the shop instead of chat message, it is a good replacement. Although, you guys should really add this feature into the API.

**Reply by Gareth064** [reply]
Hey @SannanOfficial. Etsy sellers need to be aware that if there is a dispute raised by a buyer, Etsy will not take into consideration any communications the seller had with the buyer if it wasn't done through Etsy chat. So if a buyer wasn't happy about something you sold them, and your only proof is an email, that it's their fault and not yours, then Etsy won't care and take the side of the buyer.

**Reply by SannanOfficial** [reply]
@Gareth064 I understand that. The email in this case being generated is a professional warm, and welcoming message that is supposed to be sent to the buyer after they place the order and the receipt is generated.
